src/api/notification-template/content-types/notification-template/schema.json

```diff
@@ -0,0 +1,31 @@
+{
+  "kind": "collectionType",
+  "collectionName": "notification_templates",
+  "info": {
+    "singularName": "notification-template",
+    "pluralName": "notification-templates",
+    "displayName": "Notification template",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": true
```

Comment:
this is the flag I mentioned. I would set it to false

Reply:
Fixed